### ISSUE TITLE
VM Detected

### ISSUE THREAD

**Issue by svox07** [UnamSanctam/SilentCryptoMiner#2008]
Привет Унам следующее обновление добавьте этот код для обнаружения Sandboxie.VM хоть как-то обезопасить от VirusTotal

bool VM = (new System.Management.ManagementObjectSearcher("SELECT * FROM Win32_PortConnector")).Get().Count == 0;

            if (VM)
            {
                MessageBox.Show("VM Detected");
            }
            else
            {
                MessageBox.Show("Real PC Detected");
            }

**Comment by svox07** [comment]
here is the documentation about it


https://learn.microsoft.com/en-us/windows/win32/cimwin32prov/win32-portconnector

**Comment by svox07** [comment]
![1 (1)](https://github.com/UnamSanctam/SilentCryptoMiner/assets/70852343/2b71b0af-c447-4dc4-9556-4fc953fbdbd2)
![1644324313300](https://github.com/UnamSanctam/SilentCryptoMiner/assets/70852343/7f9d0f02-4ad7-4538-bba0-6274e93d3926)
![1](https://github.com/UnamSanctam/SilentCryptoMiner/assets/70852343/3bd02474-657f-4218-9b8f-2ead520e4738)

**Comment by UnamSanctam** [comment]
Yes that detects some VMs (and some real computers, though not most). But it does not detect VirusTotal (or hardened VMs at all) as far as I can remember.
